docker/pulsar/Dockerfile

```diff
@@ -51,6 +51,7 @@ ARG UBUNTU_MIRROR=mirror://mirrors.ubuntu.com/mirrors.txt
 
 # Install some utilities
 RUN sed -i "s|http://archive\.ubuntu\.com/ubuntu/|${UBUNTU_MIRROR:-mirror://mirrors.ubuntu.com/mirrors.txt}|g" /etc/apt/sources.list \
+     && echo 'Acquire::http::Timeout "30";\nAcquire::ftp::Timeout "30";\nAcquire::Retries "3";' > /etc/apt/apt.conf.d/99timeout_and_retries \
```

Comment:
@lhotari - I think we should add a line at the end of this `RUN` directive that removes `/etc/apt/apt.conf.d/99timeout_and_retries`. I don't think there is a reason to leave this configuration in the container image for derivative images or for the container runtime.

Reply:
> I don't think there is a reason to leave this configuration in the container image for derivative images or for the container runtime.

Why not? It's a better default for everyone and won't be harmful. 
In Ubuntu, retries have been added by default for apt since Ubuntu 21.04 (Hirsute) according to https://bugs.launchpad.net/ubuntu/+source/apt/+bug/1876035 .